rmw_fastrtps_shared_cpp/src/rmw_wait.cpp

```diff
@@ -203,6 +204,12 @@ __rmw_wait(
     }
   }
 
+  if (events) {
+    for (size_t i = 0; i < events->event_count; ++i) {
+      // no-op
+    }
+  }
```

Comment:
Maybe just comment this or remove it and place a TODO?

Reply:
Commented out and added a `TODO`.